remote/src/main/scala/org/apache/pekko/remote/transport/netty/NettyTransport.scala

```diff
@@ -462,23 +462,12 @@ class NettyTransport(val settings: NettyTransportSettings, val system: ExtendedA
 
   override def isResponsibleFor(address: Address): Boolean = true // TODO: Add configurable subnet filtering
 
-  // TODO: This should be factored out to an async (or thread-isolated) name lookup service #2960
-  def addressToSocketAddress(addr: Address): Future[InetSocketAddress] = addr match {
```

Comment:
This is a public method on a public class - I'm very surprised the mima check didn't fail - and I'm still suspicious that we are not checking mima properly for our PRs. All in all, I'm not sure that this method should be removed - it should probably be deprecated first.

Reply:
Ok, let's keep it :(